### DISCUSSION TITLE
The rook cluster was deleted with one cmd?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#11873] [Q&A]
only run "kubectl delete -f deploy/examples/common.yaml",  all resources of rook was cleaned?

```
[root@rook-node01 rook]# kubectl get ns | grep rook-ceph
rook-ceph         Terminating   15d
```

**Comment by travisn** [comment]
The namespace is defined in common.yaml. Deleting the namespace is quite a destructive action. At least many of the critical resources should have a finalizer and should not be cleaned up automatically. For example, there are a couple secrets, configmaps, and the cephcluster CR that you should at least be able to recover and try to get the cluster back up with the [disaster recovery](https://rook.io/docs/rook/latest/Troubleshooting/disaster-recovery/#restoring-crds-after-deletion).

**Reply by zhucan** [reply]
Thanks，I will try it
